code/src/components/TaskStats.js

```diff
@@ -0,0 +1,32 @@
+import React from "react"
+import { Box, AppBar } from "@material-ui/core"
+import { positions, sizing } from "@material-ui/system"
+import { useSelector } from "react-redux"
+
+
```

Comment:
is the same in the last comment. if you use ESlint lib, it might mark this as error